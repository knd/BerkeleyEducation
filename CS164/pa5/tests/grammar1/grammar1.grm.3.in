if 3 then 
	if 4 then 
		8 - 8 + 4
	else
		8 * 2 / 2
else
	1