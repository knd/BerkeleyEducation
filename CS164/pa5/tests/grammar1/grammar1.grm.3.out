('if', 3, ('if', 4, ('+', ('-', 8, 8), 4), ('/', ('*', 8, 2), 2)), 1)
